airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -3086,7 +3086,7 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-hubspot:0.1.41"
+- dockerImage: "airbyte/source-hubspot:0.1.40"
```

Comment:
I guess it should be 0.1.42 instead

Reply:
thanks for noticing this. I didn't notice this when I merged master branch. fixed it